src/command_modules/azure-cli-vsts/azure/cli/command_modules/vsts/custom.py

```diff
@@ -118,14 +125,16 @@ def _call_rp_configure_cicd(
     :type resource_group_name: String
```

Comment:
The docstring for this method still has `resource_group_name` although the argument has been changed to `registry_resource_id`.